CONTRIBUTING.md

```diff
@@ -33,15 +33,15 @@ Consult the Table of Contents below, and jump to the desired section.
 
 If you are a Kong Enterprise customer, contact the Enterprise Support channels
 by opening an Enterprise support ticket on
-[https://support.mashape.com](https://support.mashape.com/).
+[https://support.konghq.com](https://support.konghq.com/).
 
 If you are experiencing a P1 issue, please call the [24/7 Enterprise Support
-phone line](https://support.mashape.com/hc/en-us/articles/115004921808-Telephone-Support)
+phone line](https://support.konghq.com/hc/en-us/articles/115004921808-Telephone-Support)
 for immediate assistance, as published in the Customer Success Reference Guide.
 
-If you are interested in becoming a Kong Enterprise customer, please visit
-https://www.mashape.com/enterprise/ or contact us at
-[sales@mashape.com](mailto:sales@mashape.com).
+If you are interested in becoming a Kong Enterprise customer, please visit 
```

Comment:
looks like a trailing whitespace? ;) 

Better configure your editor to either trim them, or even better, to parse `.editorconfig` to avoid those in the future